filebeat/module/mysql/slowlog/_meta/fields.yml

```diff
@@ -31,6 +35,55 @@
       type: long
       description: >
         The connection ID for the query.
+    - name: thread_id
+      type: long
+      description: >
+        ID of the thread executing this query.
+    - name: schema
+      type: keyword
+      description: >
+        The schema where the slow query was executed.
+    - name: current_user
```

Comment:
ECS?

Reply:
There are two users that can be different, like user and effective user. I'd keep mysql terminology at least for this one.